crowdsec-docs/unversioned/getting_started/installation/linux.mdx

```diff
@@ -0,0 +1,146 @@
+    ```
+  </TabItem>
+
+  <TabItem value="centos8">
+    ```bash
+    curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.rpm.sh | sudo bash
+    ```
+  </TabItem>
+
+  <TabItem value="amz">
+    ```bash
+    curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.rpm.sh | sudo bash
+    ```
+  </TabItem>
+
+  <TabItem value="openwrt">
+    OpenWRT packages are available in the official repositories.
+  </TabItem>
+
+  <TabItem value="cloudlinux">
+    ```bash
+    curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.rpm.sh | os=el dist=7 sudo bash
+    ```
+  </TabItem>
+</Tabs>
+
+### Install Security Engine
+
+Once the repository is added, you can install the security engine via:
+
+<Tabs
+  defaultValue="debian"
+  groupId="operating-systems"
+  values={[
+    {label: 'Debian/Ubuntu', value: 'debian'},
+    {label: 'EL/Centos7', value: 'centos7'},
+    {label: 'EL/Centos Stream 8', value: 'centos8'},
+    {label: 'Amzn Linux 2', value: 'amz'},
+    {label: 'OpenWRT', value: 'openwrt'},
+    {label: 'CloudLinux', value: 'cloudlinux'},
+  ]}>
+  <TabItem value="debian">
+    <CodeBlock className="language-bash">apt install crowdsec</CodeBlock>
+  </TabItem>
+  
+  <TabItem value="centos7">
+    <CodeBlock className="language-bash">yum install crowdsec</CodeBlock>
+  </TabItem>
+
+  <TabItem value="centos8">
+    <CodeBlock className="language-bash">dnf install crowdsec</CodeBlock>
+  </TabItem>
+
+  <TabItem value="amz">
+    <CodeBlock className="language-bash">yum install crowdsec</CodeBlock>
+  </TabItem>
+
+  <TabItem value="openwrt">
+    <CodeBlock className="language-bash">opkg install crowdsec</CodeBlock>
+  </TabItem>
+
+  <TabItem value="cloudlinux">
+    <CodeBlock className="language-bash">yum install crowdsec</CodeBlock>
+  </TabItem>
+</Tabs>
+
+
+### Install Remediation Component
+
+:::warning
+Security Engine by itself is a detection engine, it will not block anything. You need to install a Remediation Component to enforce decisions
+:::
+
+For the quick start guide we will be installing the [iptables](https://en.wikipedia.org/wiki/Iptables) firewall Remediation Component. (This may not be optimal for your environment, please refer to the [Remediation Documentation](/bouncers/intro.md) for more information)
+
+#### IPTables
+
+<Tabs
+  defaultValue="iptables_debian"
+  values={[
+    { label: 'Debian/Ubuntu', value: 'iptables_debian' ,},
+    { label: 'RHEL/Centos/Fedora', value: 'iptables_rhel', },
+  ]
+}>
+<TabItem value="iptables_debian">
+
+```bash
+sudo apt install crowdsec-firewall-bouncer-iptables
+```
+
+</TabItem>
+<TabItem value="iptables_rhel">
+
+```bash
+sudo yum install crowdsec-firewall-bouncer-iptables
+```
+
+</TabItem>
+</Tabs>
+
+---
+
+Great, you now have CrowdSec installed on your system. Within the post installation checklist you will find the next steps to configure and optimize your installation.
```

Comment:
The next button is called "Next Steps" and sends you to a page called "Post Installation Steps". We should use either of these terms here (or even make the "Next Steps" text a link for people that don't see the "next" button :smile: )